x-pack/plugins/apm/ftr_e2e/cypress_test_runner.ts

```diff
@@ -76,6 +76,7 @@ export async function cypressTestRunner({ getService }: FtrProviderContext) {
       ES_NODE: esNode,
       ES_REQUEST_TIMEOUT: esRequestTimeout,
       TEST_CLOUD: process.env.TEST_CLOUD,
+      NO_COMMAND_LOG: 1,
```

Comment:
Is this intended to be a permanent or temporary fix? If temporary can you add a comment referencing the issue that will remove the need for this.

Reply:
Good Point, it's a temporary fix unless the actually memory issue is fixed. I will check with the Cypress team if they can provide a ticket